cmd/neofs-node/config.go

```diff
@@ -88,27 +80,45 @@ const notificationHandlerPoolSize = 10
 // values. It should not store any application helpers structs (pointers to shared
 // structs).
 // It must not be used concurrently.
+//
+// REFACTOR IS NOT FINISHED.
 type applicationConfiguration struct {
 	// _read indicated whether a config
 	// has already been read
 	_read bool
 
-	LoggerCfg struct {
+	Logger struct {
 		level string
 	}
 
-	EngineCfg struct {
+	Engine struct {
 		errorThreshold uint32
 		shardPoolSize  uint32
 		shards         []storage.ShardCfg
 	}
 
-	PolicerCfg struct {
+	Policer struct {
 		maxCapacity         uint32
 		headTimeout         time.Duration
 		replicationCooldown time.Duration
 		objectBatchSize     uint32
 	}
+
+	Morph struct {
```

Comment:
We dont't comment them?

Reply:
they were not commented initially and i can't say why they are exported made them unexported